### ISSUE TITLE
Logging off while sitting on movable chair causes multiple bugs

### ISSUE THREAD

**Issue by Fogapod** [unitystation/unitystation#6771]
### Environment
- Game Version: 21060620

### Detailed description of bug

Following my experiments with chair in #6715, there are a few more things.

When someone logs out sitting on chair, weird bugs start appearing.
- multiple players can sit on that chair together
- you can pull chair while sitting on it
- you can bump into tile where player logged off and it moves chair with person visually in the opposite direction as if you bumped into them, you can keep bumping into this tile to move chair around
- visuals are inconsistent between clients. some players only see chair without logged off person

### Steps to reproduce

1. ask someone to sit on chair on harm intent and leave server
2. try to sit on chair and interact with it
3. try to pull chair while sitting on it
4. invite someone else to also sit on chair

**Comment by Fogapod** [comment]
Apparently, you don't have to go offline for most of this to happen. Chairs are just weird

**Comment by Bod9001** [comment]
yeah, buckle yourself to chair and drag it then respawn yourself then try pushing it around it's really buggy since the it  hit box stays in the same place but the visual moves around
